stable/storage-class/values.yaml

```diff
@@ -15,6 +15,15 @@ storageClass:
   # Default is true for NuoDB.
   allowVolumeExpansion: true
 
+  # PersistentVolumes can have various reclaim policies, including "Retain", "Recycle", and "Delete".
```

Comment:
Do we want to add something to the README about reclaimPolicy?

Reply:
good catch, thanks. Will add